tests-beta/android/MASVS-STORAGE/MASTG-TEST-0242.md

```diff
@@ -0,0 +1,31 @@
+---
+platform: android
+title: References to Device-Access-Security Policy APIs
+id: MASTG-TEST-0242
+apis: [Build, KeyguardManager]
+type: [static]
+weakness: MASWE-0008
+best-practices: []
+---
+
+## Overview
+
+This test checks whether an application is running on a device with secure policies such as
+
+- device passcode is set
+- running on a recent version of Android OS
+- running on a secure system build intended for the end users
```

Comment:
As discussed in Slack, these two can be removed from here after the changes done in MASWE-0008. If anything, they'd belong in MASWE-0077 or other MASWEs related to MASVS-RESILIENCE-1